src/index.html

```diff
@@ -7,8 +7,55 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="page-top">
+    <header class="header">
+      <a href="#page-top">
+        <img
+        src="images/logo.png"
+        alt="logo"
+        class="logo-link"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#apple" class="nav-link is-active">Apple</a>
+          </li>
+          <li class="nav-item">
+            <a href="#samsung" class="nav-link">Samsung</a>
+          </li>
```

Comment:
```suggestion
        <ul class="nav-list">
          <li class="nav-item">
            <a href="#apple" class="nav-link is-active">Apple</a>
          </li>
          
          <li class="nav-item">
            <a href="#samsung" class="nav-link">Samsung</a>
          </li>
```
add spaces between sibling multiline blocks